### DISCUSSION TITLE
why don't I see the home screen shown in the screenshots?

### DISCUSSION THREAD

**Discussion by ayushxx7** [NvChad/NvChad#2448] [Q&A] [answered]
When I open nvim on windows, I am greeted with a blank page ready for input. What I want is the home screen with the find files and things on the menu as shown in the screenshots.

After typing `nvim` on powershell, I get this screen:
![image](https://github.com/NvChad/NvChad/assets/16764027/189cab65-5e70-4862-afe0-887e3513269b)

**Comment by siduck** [comment]
@ayushxx7 Nvdash command and please override its config to load it on startup if u want. please read all of nvchad.com docs as well

**Comment by your-moon** [comment, marked as answer]
Open chadrc.lua file. M.ui inside you can add
`    
nvdash = {
    load_on_startup = true,
},
`
like that
